src/UI/Pages/CompileReports/Components/StatusLabel.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Label } from "@patternfly/react-core";
+
+import { CompileStatus } from "@/Core";
+
+export const StatusLabel: React.FC<{ status: CompileStatus }> = ({
```

Comment:
I think we discussed during the planning meeting that the label should be like the label in the dryrun dropdown.
So the loading spinner should be inside the label.
Maybe that component can be turned into something reusable.
I think all our current status labels are lowercase, so I think this one should be lowercase as well.